Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx

```diff
@@ -15,908 +15,907 @@
+  add(e2d, 717, 103, 848, 6, 1);
+  add(e2d, 717, 1033, 848, 1, 0);
+  add(e2d, 717, 1032, 848, 1, 1);
+  add(e2d, 717, 1031, 848, 1, 2);
+  add(e2d, 717, 1030, 848, 1, 3);
+  add(e2d, 717, 1133, 848, 3, 0);
+  add(e2d, 717, 1134, 848, 3, 1);
+  add(e2d, 717, 1135, 848, 3, 2);
+  add(e2d, 717, 1129, 848, 5, 0);
+  add(e2d, 717, 1130, 848, 5, 1);
+  add(e2d, 717, 1125, 848, 7, 0);
+  add(e2d, 717, 1126, 848, 7, 1);
+  add(e2d, 717, 308, 849, 0, 0);
+  add(e2d, 717, 307, 849, 0, 1);
+  add(e2d, 717, 306, 849, 0, 2);
+  add(e2d, 717, 305, 849, 0, 3);
+  add(e2d, 717, 304, 849, 0, 4);
+  add(e2d, 717, 211, 849, 2, 0);
+  add(e2d, 717, 212, 849, 2, 1);
+  add(e2d, 717, 213, 849, 2, 2);
+  add(e2d, 717, 214, 849, 2, 3);
+  add(e2d, 717, 215, 849, 2, 4);
+  add(e2d, 717, 206, 849, 4, 0);
+  add(e2d, 717, 207, 849, 4, 1);
+  add(e2d, 717, 208, 849, 4, 2);
+  add(e2d, 717, 201, 849, 6, 0);
+  add(e2d, 717, 202, 849, 6, 1);
+  add(e2d, 717, 203, 849, 6, 2);
+  add(e2d, 717, 1325, 849, 1, 0);
+  add(e2d, 717, 1326, 849, 1, 1);
+  add(e2d, 717, 1327, 849, 1, 2);
+  add(e2d, 717, 1243, 849, 3, 0);
+  add(e2d, 717, 1242, 849, 3, 1);
+  add(e2d, 717, 1241, 849, 3, 2);
+  add(e2d, 717, 1240, 849, 3, 3);
+  add(e2d, 717, 1234, 849, 5, 0);
+  add(e2d, 717, 1233, 849, 5, 1);
+  add(e2d, 717, 1229, 849, 7, 0);
+  add(e2d, 717, 1228, 849, 7, 1);
+  add(e2d, 718, 111, 850, 0, 0);
+  add(e2d, 718, 112, 850, 0, 1);
+  add(e2d, 718, 113, 850, 0, 2);
+  add(e2d, 718, 106, 850, 2, 0);
+  add(e2d, 718, 107, 850, 2, 1);
+  add(e2d, 718, 108, 850, 2, 2);
+  add(e2d, 718, 101, 850, 4, 0);
+  add(e2d, 718, 102, 850, 4, 1);
+  add(e2d, 718, 103, 850, 4, 2);
+  add(e2d, 718, 1139, 850, 1, 0);
+  add(e2d, 718, 1138, 850, 1, 1);
+  add(e2d, 718, 1134, 850, 3, 0);
+  add(e2d, 718, 1133, 850, 3, 1);
+  add(e2d, 718, 1129, 850, 5, 0);
+  add(e2d, 718, 1128, 850, 5, 1);
+  add(e2d, 718, 12, 851, 0, 0);
+  add(e2d, 718, 11, 851, 0, 1);
+  add(e2d, 718, 8, 851, 2, 0);
+  add(e2d, 718, 7, 851, 2, 1);
+  add(e2d, 718, 4, 851, 4, 0);
+  add(e2d, 718, 3, 851, 4, 1);
+  add(e2d, 718, 1033, 851, 1, 0);
+  add(e2d, 718, 1034, 851, 1, 1);
+  add(e2d, 718, 1029, 851, 3, 0);
+  add(e2d, 718, 1030, 851, 3, 1);
+  add(e2d, 718, 1025, 851, 5, 0);
+  add(e2d, 718, 1026, 851, 5, 1);
+  add(e2d, 719, 8, 852, 0, 0);
+  add(e2d, 719, 7, 852, 0, 1);
+  add(e2d, 719, 4, 852, 2, 0);
+  add(e2d, 719, 3, 852, 2, 1);
+  add(e2d, 719, 1029, 852, 4, 0);
+  add(e2d, 719, 1030, 852, 4, 1);
+  add(e2d, 719, 1025, 852, 6, 0);
+  add(e2d, 719, 1026, 852, 6, 1);
+  add(e2d, 719, 106, 852, 1, 0);
+  add(e2d, 719, 107, 852, 1, 1);
+  add(e2d, 719, 108, 852, 1, 2);
+  add(e2d, 719, 101, 852, 3, 0);
+  add(e2d, 719, 102, 852, 3, 1);
+  add(e2d, 719, 103, 852, 3, 2);
+  add(e2d, 719, 1134, 852, 5, 0);
+  add(e2d, 719, 1133, 852, 5, 1);
+  add(e2d, 719, 1129, 852, 7, 0);
+  add(e2d, 719, 1128, 852, 7, 1);
 }
 void fillSolar2FeeLinkCH7L(std::map<uint16_t, uint32_t>& s2f)
 {
-  add_cru(s2f, 30, 0, 728);
```

Comment:
Link ID 728 belongs to CH8L... why in the old mapping is was associated to CH7L?